code/FinanceManager.Components/Components/Dashboard/Cards/Assets/AssetsDistributionOverviewCard.razor.css

```
@@ -0,0 +1,86 @@
+    gap: 16px;
+    width: 100%;
+    height: 100%;
+    padding: 4px 12px 12px;
+    min-height: 0;
+}
+
+.ad-chart-wrap {
+    flex: 0 0 auto;
+}
+
+.ad-chart-wrap ::deep .apexcharts-canvas {
```

Stylelint (17.12.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)